packages/react-examples/src/react-charting/AreaChart/AreaChart.CustomAccessibility.Example.tsx

```diff
@@ -145,10 +145,24 @@ export class AreaChartCustomAccessibilityExample extends React.Component<{}, IAr
 
     return (
       <>
-        <label>change Width:</label>
-        <input type="range" value={this.state.width} min={200} max={1000} onChange={this._onWidthChange} />
-        <label>change Height:</label>
-        <input type="range" value={this.state.height} min={200} max={1000} onChange={this._onHeightChange} />
+        <label for="changeWidth1">change Width:</label>
```

Comment:
Instead of using 1,2,3 - add a suffix for the scenario it represents. For eg: changeWidth_acc, changeWidth_basic etc.

Reply:
Followed your inputs and fixed them.